lsp-mode.el

```diff
@@ -5666,7 +5657,7 @@ A reference is highlighted only if it is visible in a window."
        "textDocument/semanticTokens/full")
       (t "textDocument/semanticTokens"))
      `(:textDocument ,(lsp--text-document-identifier)
-       ,@(if region (list :range (lsp--region-to-range (car region) (cdr region))) '()))
+                     ,@(if region (list :range (lsp--region-to-range (car region) (cdr region))) '()))
```

Comment:
This line should be changed back, emacs indentation is wrong here

Reply:
I edited the `.dir-locals.el` to use different plist indentation in the `lsp-mode` repo. If it seems fine to you then I'll make the plist changes elsewhere also.